swagger.yaml

```diff
@@ -2089,6 +2115,29 @@ definitions:
             hostname:
               type: string
 
+  day2-cluster-create-params:
+    type: object
+    required:
+      - id
+      - name
+      - api_vip_dnsname
```

Comment:
The var is generated to 'APIVipDnsname'. Maybe 'api_vip_dns_name ' would be clearer (i.e. APIVipDnsName)?